erts/etc/unix/cerl.src

```diff
@@ -81,8 +81,27 @@ run_asan=no
 run_rr=no
 skip_erlexec=no
 
+prog="$0"
+progdir="`dirname ${prog}`"
```

Comment:
This way to find the installation directory tends to break if a user has placed a symlink to the binary in the `$PATH` (a use case which worked when the installation path was hard-coded in the script). In a program I maintain I use:
```
READLINK="readlink"
if command -v greadlink > /dev/null; then
  READLINK="greadlink" # Prefer greadlink over readlink
fi

if ! $READLINK -f "$(which "$0")" > /dev/null 2>&1 ; then
    ROOT="$(dirname "$(which "$0")")"
    if [ ! -f "$ROOT/hg-fast-export.py" ] ; then
        echo "hg-fast-exports requires a readlink implementation which knows" \
             " how to canonicalize paths in order to be called via a symlink."
        exit 1
    fi
else
    ROOT="$(dirname "$($READLINK -f "$(which "$0")")")"
fi
```
The `greadlink` vs `readlink` hack is apparently needed on OSX .

Reply:
In [Wings3D](http://www.wings3d.com) we use the following trick on macOS to get absolute path to root directory:
```
pushd `dirname $0` > /dev/null
ROOTDIR=`pwd -P`
popd > /dev/null
```

Reply:
I've added a testcase that test that it works to if you do a symlink to `erl` and execute that it still works. I don't really see any point in fixing `cerl` as that is a debug tool meant to be in the source tree, no need to make it more complex than it has to be.